matrix-sdk-android/src/main/java/im/vector/matrix/android/internal/database/model/EventEntity.kt

```diff
@@ -59,6 +59,8 @@ internal open class EventEntity(@Index var eventId: String = "",
             sendStateStr = value.name
         }
 
+    var ageLocalTs: Long? = null
```

Comment:
Why it is not declared in the constructor?

Reply:
moved it up